test/integrationTest/articles.test.js

```diff
@@ -1,9 +1,8 @@
 import chai from 'chai';
 import chaiHttp from 'chai-http';
 import app from '../../src/app';
-import { UserController, ArticleController } from '../../src/controller';
+// import { UserController, ArticleController } from '../../src/controller';
```

Comment:
Also, you should consider removing the  commented line of code if it would not be required in the implementation